src/tls.rs

```diff
@@ -25,6 +25,18 @@
 // NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+#[cfg(windows)]
+extern crate winapi;
+#[cfg(windows)]
+use winapi::um::wincrypt::{
+    CertCloseStore,
```

Comment:
Nit: generally we don't group imports like this, we simply expand all the `use`s.

Reply:
removed all imports/usage